frontend/templates/troubleshooting/index.tsx

```diff
@@ -130,126 +110,73 @@ const Wiki: NextPageWithLayout<{
    return (
       <>
          <GoogleNoScript />
+         <TagManager />
+         <Metadata
+            metaDescription={metaDescription}
+            metaKeywords={metaKeywords}
+            canonicalUrl={canonicalUrl}
+            title={title}
+         />
+         <HreflangUrls urls={wikiData.hreflangUrls} />
+
          <NavBar
             editUrl={wikiData.editUrl}
             historyUrl={wikiData.historyUrl}
             deviceGuideUrl={wikiData.deviceGuideUrl}
             devicePartsUrl={wikiData.devicePartsUrl}
             breadcrumbs={wikiData.breadcrumbs}
          />
+
          <TOCContextProvider defaultItems={tocItems}>
-            <Flex>
+            <Box
+               className="layout-grid"
+               display="grid"
+               sx={{
+                  gridTemplateColumns: {
+                     base: '[toc] none [wrapper] 1fr',
+                     lg: '[toc] auto [wrapper] 1fr',
+                  },
+               }}
+               ref={contentContainerRef}
+            >
                <TOC
+                  className="summary"
                   contentRef={contentContainerRef}
-                  flexShrink={{ lg: 0 }}
-                  flexGrow={1}
                   borderRight={{ lg: '1px solid' }}
                   borderColor={{ lg: 'gray.300' }}
-                  maxWidth={{
-                     base: 'calc(100% + 2 * var(--chakra-space-4))',
-                     lg: '240px',
-                  }}
-                  marginLeft={-4}
-                  marginRight={{ base: -4, lg: 0 }}
-                  listItemProps={{
-                     paddingLeft: { lg: 4 },
-                  }}
+                  maxWidth={{ base: 'calc(100% + 2 * var(--chakra-space-4))' }}
+                  listItemProps={{ paddingLeft: { lg: 4 } }}
+                  gridArea="toc"
+                  width="220px"
                />
-               <Container
+               <Stack
+                  className="wrapper"
+                  direction={{ base: 'column', xl: 'row' }}
                   fontSize="md"
                   maxW="1280px"
-                  display="flex"
-                  flexWrap={{ base: 'wrap', lg: 'nowrap' }}
-                  ref={contentContainerRef}
```

Comment:
@jarstelfox does this `ref` need to be added back in? Worried this got lost in the shuffle.

Reply:
Yes, it does or else the mobile TOC will not display. We hide it unless we reach the main content

Reply:
Ahhh, there it is. It just got moved up to the parent: https://github.com/iFixit/react-commerce/pull/1988/files#diff-6ee89f2292a5e40be6686c92372cc0dbe1f2b29af769f28390b2a61061fe1008R140